android/src/main/java/com/azure/reactnative/notificationhub/ReactNativeFirebaseMessagingService.java

```diff
@@ -121,4 +123,13 @@ private void runBackgroundTask(Context context, Bundle bundle) {
             Log.w(TAG, "No task name");
         }
     }
+
```

Comment:
Below seems to a whitespace change or something like that? Maybe it should be excluded from the PR?